lesson02/assignment/css/_font.variables.scss

```diff
@@ -0,0 +1,8 @@
+//font-sizes
+$small: 1.000em;
+$medium: 1.250em;
+$large: 1.625em;
+
+//font-styles
+$italic: italic;
+$center: center;
```

Comment:
This is an excellent use of variables. For a large site, having font information centralized like this makes sitewide style adjustments quick and easy.

Reply:
Thank you! It looks much nicer in the code, too.